OsuPlayer/Views/StatisticsViewModel.cs

```diff
@@ -81,31 +93,35 @@ public StatisticsViewModel()
 
     private async void Block(CompositeDisposable disposable)
     {
+        Disposable.Create(() => { _timer?.Close(); }).DisposeWith(disposable);
+
         // Done separately so they can fail independently
         var tasks = new[]
         {
-            UpdateApiStatistics(),
-            UpdateBeatmapCount(),
-            UpdateStorageAmount()
+            UpdateApiStatistics(), UpdateBeatmapCount(), UpdateStorageAmount()
         };
 
         await Task.WhenAll(tasks);
 
         UpdateDate();
+
+        _timer = new Timer(1000);
+        _timer.Elapsed += (sender, args) => UpdateDateNonApi();
+        _timer.Start();
     }
 
     private void UpdateDate()
     {
-        var timeSince = DateTime.Now.Subtract(new DateTime(2017, 11, 1));
+        _playerAgeTime = DateTime.Now.Subtract(new DateTime(2017, 11, 1));
 
-        var time = DateTime.MinValue + timeSince;
+        this.RaisePropertyChanged(nameof(PlayerAge));
+    }
 
-        // note: MinValue is 1/1/1 so we have to subtract...
-        var years = time.Year - 1;
-        var months = time.Month - 1;
-        var days = time.Day - 1;
+    private void UpdateDateNonApi()
+    {
+        _playerAgeTime += TimeSpan.FromSeconds(1);
```

Comment:
Maybe instead of simply adding one second, why not just call `UpdateDate()` to refresh it to the actuall value?
Or did you tried that out and it worked wonky?

Reply:
It worked but I wanted to reduce API calls, calling the API for a date which can be updated like this seems pretty wasted to me.

Reply:
`UpdateDate()` doesn't call the API at all
```CSharp
private void UpdateDate()
{
    _playerAgeTime = DateTime.Now.Subtract(new DateTime(2017, 11, 1));

    this.RaisePropertyChanged(nameof(PlayerAge));
}
```

Reply:
Oh wait, i mistook something, this does not call the api even xD

Reply:
I'll fix that